suites/apis/requestorTest.js

```diff
@@ -0,0 +1,84 @@
+Feature('RequestorAPI');
+
+/**
+*   Create an access request for a policy that doesn’t exist in Arborist => 400 error.
+
+*   The user does not have access. Create an access request for a policy that exists in Arborist.
+    Update the access request status. The user should now have access.
+    For this test, we need a 2nd user that has access to update statuses in Requestor;
+    this is already set up since it’s also needed for the existing Study Viewer tests.
+
+*   The user has access. Create a “revoke” access request. Update the access request status.
+    The user should not have access anymore.
+
+ */
+
+const { expect } = require('chai');
+const requestorTasks = require('../../services/apis/requestor/requestorTasks.js');
+
+// Create an access request for a policy that doesn’t exist in Arborist => 400 error
+Scenario('User requests access for a policy that does not exist in Arborist @requestor', async ({
+  home, login, users,
+}) => {
+  home.do.goToHomepage();
+  login.complete.login(users.user0);
+  const response = await requestorTasks.createRequestFromPolicyID(users.user0, 'random-policy');
+  expect(response).to.have.property('status_code', 400);
+});
+
+Scenario('User does not have access request for a policy and requests for access followed by sending a Revoke request @requestor', async ({
```

Comment:
maybe make the scenario name shorter? otherwise i like the scenario and breaking down of the steps with the addition of checking that the access is not granted before it's signed 😄